README.md

```diff
@@ -1,158 +1,53 @@
 
-##### Prototyp interfejsu użytkownika
-Przygotowanie projektu od 0 lub implementacja bez projektu.
+### Wykorzystane technologie:
 
-##### Wymagania funkcjonalne:
-1. Administrator wprowadza informację o placówkach w sieci kin. Sieć ma zamiar posiadać Kina w różnych miastach na całym świecie. 
-Dlatego konieczne jest, aby prowadzić ewidencję Kin sieci. Kino musi mieć swój unikalny numer identyfikacyjny. Każda placówka ma określony adres i godziny otwarcia.
-1. Administrator wprowadza informacje o filmie, który będzie dostępy dla wszystkich placówek w sieci.
-1. Usunięcie filmu powinno skutkować odwołaniem wszystkich zaplanowanych projekcji, anulowaniem rezerwacji i wysłaniem do klientów, którzy zakupili bilety, wiadomości o zwrocie kosztów.
-1. Administrator wprowadza informacje o seansach w danych kinie i przypisuje do seansu salę kinową i film oraz określa godzinę.
-1. W danej sali kinowej nie mogą odbywać się 2 seanse jednocześnie. Dodatkowo między seansami należy zachować co najmniej 15 minutową przerwę na sprzątanie.
-1. Seans musi się kończyć co najmniej 15 minut przed zamknięciem kina i zaczynać minimum 15 minut po otwarciu kina.
-1. Każda sala kinowa ma określone, ile posiada kolumn i rzędów (zakładamy uproszczony model prostokąta).
-1. Administrator zarządza listą cen za bilety. Wybiera jakie bilety są dostępne na które seanse. (np. na seans premierowy nie obowiązują zniżki, ale na resztę można zakupić bilet studencki).
-1. Administrator otrzymuje raport w czasie rzeczywistym o liczbie sprzedanych i zarezerwowanych miejsc. Dzięki temu będzie mógł podejmować
-decyzje, które zwiększa przychód.
-1. Administrator może konfigurować czy na dany seans można zostawiać (w czasie zakupu lub rezerwacji) puste miejsca na rogu rzędów.
-Tę zasadę może zmienić w każdym momencie.
-1. Administrator może włączyć specjalny tryb pandemii, w którym muszą być przynajmniej 2 miejsca odstępu w jednym rzędzie, między miejscami
-z 2 różnych rezerwacji / zakupów.
+- NodeJs / NestJS
+- MongoDB / Mongoose
+- Typescript
 
+### Uruchomienie projektu
 
-## Możliwe usprawnienia i dodatkowe funkcjonalności:
-1. Integracja z zewnętrznym systemem płatności — np. PayU (wykorzystać Sandbox, który umożliwia testowanie płatności bez prawdziwych transakcji)
-1. Sprawdzanie biletów — generowanie kodu QR z zakupionym biletem. Bilet jest „sprawdzony” po zeskanowaniu kodu QR telefonem.
-1. Kreator do ustawiania miejsc na sali w bardziej skomplikowany sposób niż wspomniany prostokąt.
+1. Zainstaluj MongoDB
+1. Baza danych jest dostępna pod adresem localhost:27017/
+1. Stwórz bazę o nazwie dance_school
+1. Zainstaluj zależności: npm install
+1. Uruchom serwer developerski: npm run start:dev
+1. Aplikacja jest dostępna pod adresem localhost:3000/
+1. Swagger jest dostępny pod adresem localhost:3000/api/
+1. Uruchomienie testów: npm run test:e2e
```

Comment:
tu fajnie by było jakby skrypty i adresy były w ``, wtedy to się ładnie formatuje i od razu rzuca w oczy ;) a nazwę bazy danych można np. kursywą (to chyba w gwiazdkach się pisze)